### DISCUSSION TITLE
MEng%40UCLA/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#102] [Announcements]
# MEng%40UCLA/

https://csmsapp.org/MEng%40UCLA/

**Comment by yyong120** [comment]
UCLA 官网说 2022年6月10号 之后的托福家考成绩不算了，好坑啊。

**Reply by yyong120** [reply]
23fall又说可以接受托福家考，奇怪的 UCLA

**Comment by Yangxinyee** [comment]
官网写的口语预期分数是24，意思是严卡口语24吗？

**Reply by yyong120** [reply]
应该不是严卡，我口语23也申上了

**Reply by Yangxinyee** [reply]
我口语也是23。然后我发邮件问了，他回复的意思是没达到最低要求的话会要求进校参加英语测试？你口语23录取之后有要求参加考试吗？
回复原文：
You are strongly encouraged to improve your language ability and submit scores that meet UCLA’s minimum requirements. If you are an admitted applicant who has not met UCLA’s English language requirement for waiving the English Requirements for this examination, you must, upon arrival at UCLA, take the UCLA English as a Second Language Placement Examination (ESLPE). The ESLPE is an English diagnostic test. Depending on your results on the ESLPE, you may be required to complete an English as a Second Language course beginning in your first term at UCLA.

**Reply by Yangxinyee** [reply]
感觉这个回复太笼统了。。。也没解释所谓的预期分数是不是最低分数线。

**Reply by yyong120** [reply]
我最后没去，在录取发 offer 阶段没收到要求参加英语测试的通知。

**Reply by Yangxinyee** [reply]
哦哦好的，谢谢回复。那能问一下bg嘛。。。想评估一下自己，麻烦啦。

**Reply by yyong120** [reply]
https://cloud.seatable.cn/dtable/external-links/ff48695db50e48358d5b/?tid=YrbT&vid=0000&row-id=WemXgI3ORkKqVJrv-K8QvA

**Reply by Yangxinyee** [reply]
谢谢哥！

**Comment by jimlue530** [comment]
今年申請體感最差的就是這個項目了，首先申請費用最貴，是別人的2-3倍 (155usd)，送出後一直沒有顯示收到T成績，寄了十幾封給program接口，沒有任何一封回覆。最後突然收到一封毫無內容的拒信 (其他ad包括CMU, UCSD等供參)。說實話上了也不會去，這種一年program突然不給defer一學期真的天會塌，而且也不給轉track。
